### PR TITLE
Update to the topic 2.2

### PR DESCRIPTION
**Pull request by Gowtham-GitHub**

### CODE REVIEW

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md [hunk 1]
```diff
@@ -21,9 +21,9 @@
     - Provides a non-AWS user with temporary AWS access by linking users Active Directory credentials
     - Uses SAML (Security Assertion markup language)
     - Allows Single Sign On (SSO) which enables users to log in to AWS console without assigning IAM credentials
-  - ***Federation with third party providers / Cognito***
-    - Used mainly in web and mobile applications
-    - Makes use of Facebook/Google/amazon etc. to federate them
+  - ***Federation with third party providers (Cognito & Web Identity Federation)***
+    - Cognito: Used mainly in web and mobile applications
+    - Web Identity Federation: Makes use of Facebook/Google/Amazon etc. to federate them
```
Comment (undergroundwires) [inline saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md:26]: Great attention to detail and pretty good to mention Web Identity Federation here. Awesome!

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md [hunk 2]
```diff
@@ -59,12 +59,12 @@
       - Cognito returns Cognito ID with temporary AWS credentials
   - E.g. Provide (temporary) access to write to S3 bucket using Facebook Login
   - 💡 Good for authenticating B2B & B2C users for web + mobile applications.
-- ***Web Identity Federation*** is an alternative to using Cognito but AWS recommends against it.
+- **Web Identity Federation** is an alternative to Cognito but AWS recommeds Web Identity Federation only for advanced scenarios. Mostly try to use Cognito, because it does most of the behind-the-scenes work with public identity provider services for you.
```
Comment (undergroundwires) [inline saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md]: Small typo: recommends. I like to point of the differences and much better description.

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md [hunk 3]
```diff
@@ -59,12 +59,12 @@
...
 
 ## AWS Single Sign On (SSO)
 
 - Centrally Managed Single Sign On across multiple AWS Accounts and Business Applications (Office 365, SalesForce, Box)
-- One login gets you access to everything securely
+- One login gets you access to management console securely
```
Comment (undergroundwires) [inline saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md:70]: 👍

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md [hunk 4]
```diff
@@ -79,7 +79,7 @@
 
 ## AWS Directory Service
 
-- AWS managed Active Directory
+- **AWS managed Active Directory**
```
Comment (undergroundwires) [inline saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md:85]: 👍

**Review by undergroundwires — COMMENTED** [review]
Great. The difference and use cases between Cognito & Web Identity Federation was not clear, now it is 😎 There's just a small typo in line 62.

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 # Identity federation & SSO
 
-- Federation lets users outside of AWS to assume temporary role (using STS) for accessing AWS resources without having a user.
+- Federation lets users outside of AWS to assume temporary role (using STS) for accessing AWS resources without having to create a user in aws.
```
Comment (undergroundwires) [inline saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md]: We should type AWS with capital letters ⬆️

Comment (Gowtham-GitHub) [reply saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md]: Agree with you.

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md (hunk 2 above)
Comment (Gowtham-GitHub) [reply saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md]: That's great...

File: saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md (hunk 1 above)
Comment (Gowtham-GitHub) [reply saa/2.2. Security - Identity federation (STS, SAML, Cognito, SSO, Directory Service).md:26]: 👍

**Pull request merged**